src/packages/README.md

```diff
@@ -26,33 +90,76 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
 
 **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
 employ fine-grained C/C++ memory protection.
-There are currently over 8,000 CheriABI packages available.
-These packages are considered *experimental*.
 
-These packages are installed in the standard `/usr/local` hierarchy.
+These packages are considered *appropriate for experimental use*.
+Their primary function is to provide necessary dependencies for efforts to port
+software to CheriABI and to support CHERI demonstrations.
+They are suitable for research and development of software to provide spatial
+and temporal memory safety as well as software compartmentalisation.
```

Comment:
Hmm. Do we mean “provide”, “use”, or “benefit from”?

Reply:
"benefit from" sounds good.